pkgs/development/libraries/gl2ps/default.nix

```
@@ -23,9 +24,10 @@ stdenv.mkDerivation rec {
 
   buildInputs = [
     zlib
+    libpng
+  ] ++ lib.optionals stdenv.isLinux [
```

Let's aim to have working Mesa on other KMS/DRM OSes.

```suggestion
  ] ++ lib.optionals (!stdenv.isDarwin) [
```

---

Done.

---

Thanks! Same goes for the other PRs doing the same change to other packages.